### PR TITLE
[SCAL-77376]: Continuous deployment via github actions

### PR DESCRIPTION
**Pull request by sumit-thoughtspot**

### CODE REVIEW

File: .github/workflows/deploy.yml [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+      # Runs linter
+      - name: Run linter
+        run: npm run lint
+
+      # Runs tests
+      - name: Run tests
+        run: npm test
+
+      # Collect coverage report
+      - uses: 5monkeys/cobertura-action@master
```
Comment (ayonghosh) [inline .github/workflows/deploy.yml:44]: I don't think we need to run the SDK specific stuff while publishing docs

Comment (sumit-thoughtspot) [reply .github/workflows/deploy.yml]: Done.

File: .github/workflows/deploy.yml [hunk 2]
```diff
@@ -0,0 +1,48 @@
+# This is a basic workflow to help you get started with Actions
```
Comment (ayonghosh) [inline .github/workflows/deploy.yml:1]: Remove comment

File: .github/workflows/deploy.yml [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+
+name: Release
```
Comment (ayonghosh) [inline .github/workflows/deploy.yml:3]: Should we name this DeployDocs?

**Review by ayonghosh — APPROVED** [review]
LGTM after addressing comments

File: .github/workflows/deploy.yml (hunk 2 above)
Comment (sumit-thoughtspot) [reply .github/workflows/deploy.yml]: Done.

**Pull request merged**